lua/nvim-lsp-installer/ui/status-win/init.lua

```diff
@@ -233,7 +233,7 @@ local function init(all_servers)
     local window = display.new_view_only_win "LSP servers"
 
     window.view(function(state)
-        return Ui.Node {
+        return Indent {
```

Comment:
Wrapping the entire view with an `Indent` root node here, which effectively has the same effect as the change in initial indentation